llama/cli.py

```diff
@@ -12,6 +13,8 @@
 @click.group()
 @click.pass_context
 def cli(ctx):
+    if config.SENTRY_DSN:
+        sentry_sdk.init(dsn=config.SENTRY_DSN)
```

Comment:
For some reason this didn't work for me as a kwarg, I had to change it to `sentry_sdk.init(config.SENTRY_DSN)`. I would also pass `environment=config.ENV` to `sentry_sdk.init()` so we know which env the exceptions occur in. Sentry will let you look at and set alerts for specific envs so we don't get inundated with dev exceptions if someone forgets to unset their local SENTRY_DSN env variable :)